### DISCUSSION TITLE
Implement bulk packaging from CLI

### DISCUSSION THREAD

**Discussion by merelcht** [kedro-org/kedro#3855] [Idea]
(transfer from Jira, created by @lorenabalan)

Enable packaging multiple modules in one CLI command, i.e. kedro micropkg package p1,p2,p3

Be mindful of how this interacts with the other options, --alias and --destination:
 - leave --destination to work for all module mentioned in the command
 - make --alias actually be encoded in the command, like {{kedro module package p1:new_p1,p2:new_p2 }}(breaking change or new feature?)

Maybe, if agreed/validated, also something like kedro module package p1 --save (package modules and update pyproject.toml), similar to npm packaging.

**Comment by astrojuanlu** [comment]
Unlinking this from the micropackaging functionality and turning it into a discussion.
